Jamroot

```diff
@@ -149,7 +151,8 @@ constant BOOST_VERSION : 1.85.0 ;
 constant BOOST_JAMROOT_MODULE : $(__name__) ;
 
 # Allow subprojects to simply `import config : requires ;` to get access to the requires rule
-modules.poke : BOOST_BUILD_PATH : $(BOOST_ROOT)/libs/config/checks [ modules.peek : BOOST_BUILD_PATH ] ;
+import-search $(BOOST_ROOT)/libs/config/checks ;
+import-search $(BOOST_ROOT)/libs/predef/tools/checks ;
```

Comment:
Does this still work if `libs/predef` is empty? (In a partial checkout)

Reply:
I actually do remember an occasion on which `libs/config` was empty and things failed somewhere, which is why I was reminded to ask. :-)

(That's why depinst always installs `libs/config`.)

Reply:
(But `libs/config` is needed for more than that - the architecture and address model checks are there. Which is another story; these aren't Boost-specific and need to be in b2 proper.)